### PR TITLE
Dumbpad VIA support added

### PR DESCRIPTION
**Pull request by jfescobar18**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [x] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by jfescobar18** [comment]
This keymap adds VIA support for this keyboard

**Review by drashna — APPROVED** [review]

File: keyboards/dumbpad/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+}
+
+void keyboard_post_init_user(void) {
+    // Customise these values to desired behaviour
+    //debug_enable = true;
+    //debug_matrix = true;
+    //debug_keyboard = true;
+    //debug_mouse = true;
+}
+
```
Comment (fauxpark) [inline keyboards/dumbpad/keymaps/via/keymap.c:112]:
```suggestion
```

File: keyboards/dumbpad/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+}
+
+void keyboard_post_init_user(void) {
+    // Customise these values to desired behaviour
+    //debug_enable = true;
+    //debug_matrix = true;
+    //debug_keyboard = true;
+    //debug_mouse = true;
+}
+
```
Comment (fauxpark) [inline keyboards/dumbpad/keymaps/default/keymap.c:74]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by drashna** [comment]
This needs to be rebased to fix a bunch of the unintentional commits added to the PR.

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**